client/coral-framework/graphql/mutations/index.js

```diff
@@ -171,3 +172,22 @@ export const stopIgnoringUser = graphql(STOP_IGNORING_USER, {
     };
   }
 });
+
+export const editComment = graphql(EDIT_COMMENT, {
+  props: ({mutate}) => {
+    return {
+      editComment: (id, asset_id, edit) => {
+        return mutate({
+          variables: {
+            id,
+            asset_id,
+            edit,
+          },
+          refetchQueries: [
+            'EmbedQuery'
```

Comment:
You'd have to update the query using the mutation result and the optimistic response. Refetching is not only ineffective (extra round trip, server load, and slow ui response) but also removes any pagination data that was fetched from the server and merged into the query.